### DISCUSSION TITLE
Watch specific discussion category for repo

### DISCUSSION THREAD

**Discussion by huehnerlady** [community/community#3951] [Discussions]
Hi,

subscribing to github/feedback I found out that it is quite verbose if you cannot filter the discussions type you want to watch.
In my ase eg. I am just interested in the `Discussions Feedback`, but I get notifications for all new discussions. I think it also makes sense in general. A lot of Repos now have the `Q&A` discussions, so maybe you want to help out and just get notified if someone has a question, but not everytime there is a new feedback :) 

So my idea is that you not just have the tick box `Discussions`, but maybe one you tick discussions, that you get all the available categories and then you can ask for the categories you want to subscribe. 

Discussions subscription
![image](https://user-images.githubusercontent.com/12533979/118271608-cd3dcf80-b4c1-11eb-8434-c69bb19c23c3.png)

**Comment by gr2m** [comment]
I'd love that, too. My usecase is to subscribe to only the new "Announcements" category in repositories, e.g. https://github.com/johnlindquist/kit/discussions/categories/announcements

**Comment by alliepiper** [comment]
Same here. We need a way to connect a mailing list to only a single category, and this doesn't look possible at the moment.

**Comment by johnkw** [comment]
Yikes - this is extremely important. A repo previously only had Announcements and now added another discussion group, and there's no way to unsubscribe from the new group, so you're forced to unsubscribe from both.

**Reply by huehnerlady** [reply]
I would actually bring up the github feedback discussions as the best example: There are so many categories now and I am actually just interested in one category. But because I can just watch one category, I get tons and tons of notifications. So I have to unsubscribe, but then I have to unsubscribe from all, which is a real shame as I am interested in the progress of GitHub

**Comment by cirosantilli** [comment]
Related for labels: https://github.com/isaacs/github/issues/1172

**Comment by pattacini** [comment]
Now that GitHub has aggregated even more feedback services into this Discussion platform, this has become a super urgent need.

**Comment by jfullstackdev** [comment]
yup nice suggestion, as for now, there is the Category and Labels that you can simply bookmark to be as one click away

**Comment by bumbleshoot** [comment]
This is badly needed! I am announcing new versions, bug fixes, etc. for my project, but my users have no way of knowing about my announcements. The announcements feature is essentially useless until this feature is implemented.

**Comment by alexandercbooth-test** [comment]
test

**Comment by CanRau** [comment]
Yes please 🙏

I also had some occasions where I only wanted to subscribe to one answer and its replies in a discussion and not necessarily the whole discussion 😇

For example I could subscribe to only replies to this comment of mine instead of the whole discussion 🤓

❤️

**Reply by gr2m** [reply]
> For example I could subscribe to only replies to this comment of mine instead of the whole discussion 🤓

I'd file this as a separate discussion, it's out of scope for watching a specific category

**Reply by CanRau** [reply]
Good point 👍 While almost opening a new discussion I stumbled upon [#12952](https://github.com/orgs/community/discussions/12952)

**Reply by Preetiraj3697** [reply]
what meaning of license on repo?

**Comment by jayceeday** [comment]
Hi everyone!  I'm Jaycee, Designer on GitHub Discussions and I have started to explore this idea. Thank you @huehnerlady and everyone else for the great feedback!

Wanted your input, is it something like this that you would like? This would subscribe to everything in that category: 

<img width="1102" alt="Screenshot 2022-10-14 at 17 42 22" src="https://user-images.githubusercontent.com/86061938/195874664-bcb39a5a-db7c-415f-ab1b-151daaec68f7.png">

Let me know your thoughts, or if you need something more granular.

Thanks!

**Reply by johnkw** [reply]
That's good enough, although I'd rather have little enabled or disabled "mailbox/letter" icons on the left next to each category, so you can immediately see which ones you've subscribed to.

**Reply by pointlessone** [reply]
Yes, please. This would cover the release announcements case.

One addition I'd love is an RSS/Atom feed for people to subscribe in a reader if they don't want to get emails. Just for the discussion top posts in that category.

**Reply by gr2m** [reply]
This looks great, and consistent with subscribe/unsubscribe UI in other places on github.com 😍

**Reply by andrew222651** [reply]
I'd benefit from being able to subscribe to labels not just categories.

**Reply by jayceeday** [reply]
Great, thank you all!

@johnkw that's a great idea, I will explore this. 

@pointlessone thanks! There was a [separate request for this here](https://github.com/orgs/community/discussions/31) if you want to vote/add comments/see what other people have done for this. I will take a look at this though and see if we can add this.

@amacfie you can subscribe to labels by going to the repo code page and clicking watch > labels and filter by label. If you mean something different please look for an open feedback issue or create a new one?
<img width="1600" alt="Screenshot 2022-10-17 at 11 15 58" src="https://user-images.githubusercontent.com/86061938/196124814-4b6b4fd9-6e6c-457c-b963-5aec6cef17ae.png">

**Reply by yoannchaudet** [reply]
This looks great. Really looking forward to using it to monitor the community better!

**Reply by yashi** [reply]
@jayceeday Is "Filter by labels" for "issues" available for all users?  I don't see it when I click on "watch" in this repo or the others.

**Reply by jasonpincin** [reply]
This would be extremely useful to us! Wanted to up-vote this!

**Reply by ceolinwill** [reply]
@jayceeday Is there a way to follow the progress on this feature or get notified when it's released?

**Reply by koozz** [reply]
An alternative would be to have the same "Watch > Custom > Issues > Filter by label"-feature be implemented for the discussions, so "Watch > Custom > Discussions > _*Filter by label*_". That is a different scope than an actual category, but it already comes a long way and a comparable feature already exists.

**Reply by sebastianhaeni** [reply]
Category subscription is nice. But label based subscriptions on newly created discussions or when the label is added would be a feature we need.

So if only categories can be subscribed to or not, we would need to explode our current `Q&A` category into multiple categories (e.g. `Q&A-backend`, `Q&A-cicd`, etc.) That would not be so nice.

**Comment by Birdza87** [comment]
.

**Comment by EricZinda** [comment]
Any updates on this feature?  It is sorely needed

**Reply by michaelnguyenaa** [reply]
bump

**Comment by chris-wickens** [comment]
@jayceeday your proposed solution looks ideal from my perspective. Do you have an update to share?

**Reply by rdtechie** [reply]
@jayceeday Any updates?

**Comment by kristianw** [comment]
Any updates? It's been a while now since we last heard anything

**Comment by stella-bourdin** [comment]
Hi, 
I'd really like this feature as well to be able to send email announcements to the users of my package without creating a mailing list... 
You suggestion @jayceeday looks perfect to me, I do not even need label-based. Could you please update on the advances of this? 
Regards, 
Stella

**Comment by mbenkmann** [comment]
I want this, too!

**Reply by diogo-vf** [reply]
same !

**Comment by BeanBagKing** [comment]
@jayceeday I think that looks good. I see a lot of people here want to only subscribe to a specific discussion category. In my case I want to exclude one specifically and subscribe to all of the others. There is a "daily builds" that has been setup for one repo, I don't need to be notified every time, but I'd like to keep tabs on the rest. It looks like any method that would allow you to subscribe/unsubscribe on a per-category basis would work.
